azure-communication-ui/azure-communication-ui/build.gradle

```diff
@@ -9,13 +9,13 @@ apply from: file('publish.gradle')
 apply from: file('jacoco.gradle')
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
```

Comment:
Adam, we can do this in separate PR. We need to update build tools to version 31 for this change.

Reply:
It's required for this PR, so it'll have to be merged first.